Resources/config/routing.yml

```diff
@@ -122,6 +122,10 @@ canal_tp_mtt_block_delete:
         externalNetworkId: -?[a-zA-Z0-9_\-:]*
 
 #Calendar
+canal_tp_mtt_calendar_create:
+    pattern: /calendars/create
```

Comment:
pattern is deprecated, use path instead

https://github.com/symfony/symfony/pull/6738/files#diff-0